crawl-ref/source/spl-summoning.cc

```diff
@@ -817,13 +817,16 @@ int mons_ball_lightning_hd(int pow, bool random)
     return ball_lightning_hd(pow, random) / 2;
 }
 
-spret cast_conjure_ball_lightning(int pow, god_type god, bool fail)
+spret cast_conjure_ball_lightning(const actor &agent, int pow, god_type god, bool fail)
```

Comment:
You don't need to add an agent here. This is only ever used by the player. Monster conjure ball lightning uses a completely different codepath - see https://github.com/crawl/crawl/blob/master/crawl-ref/source/mon-cast.cc#L5959 .

Reply:
thanks! removed the agent from parameter.